security/proto/istioca.proto

```diff
@@ -0,0 +1,49 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package istio.v1.auth;
+
+import "gogoproto/gogo.proto";
```

Comment:
just curious - these lines are required ? seems not in https://github.com/istio/istio/blob/collab-gcp-identity/security/proto/ca/v1alpha1/istioca.proto

Reply:
They are not required as far as I see for now. I removed them.